README.md

```diff
@@ -134,6 +134,7 @@ The following table shows the supported combinations:
 
 | LiT              | LND          | Loop         | Faraday       | Pool         |
 |------------------|--------------|--------------|---------------|--------------|
+| **v0.9.2-alpha** | v0.16.2-beta | v0.23.0-beta | v0.2.11-alpha | v0.6.2-beta  |
```

Comment:
nit: needs an entry in the above compatibility table as well.

Reply:
added 👍